Gemfile

```diff
@@ -1,4 +1,6 @@
-source "https://rubygems.org"
+# frozen_string_literal: true
+
+source 'https://rubygems.org'
```

Comment:
Rubocop should probably best be mounted on CI, but I can open another PR for that

Reply:
Yes, let's do that in separate pull request.